packages/trader/src/Modules/MT5/Containers/mt5-dashboard.jsx

```diff
@@ -80,11 +81,18 @@ class MT5Dashboard extends React.Component {
             beginRealSignupForMt5,
             createMT5Account,
             is_loading,
+            landing_company_shortcode,
             has_mt5_account,
             has_real_account,
             NotificationMessages,
+            is_logged_in,
+            can_upgrade_to,
         } = this.props;
 
+        if (is_logged_in && can_upgrade_to !== 'svg' && landing_company_shortcode !== 'svg') {
```

Comment:
maybe we can have config value is_metatrader_eligible for this? if not, just make a named variable for this one